### PR TITLE
capstone project personal blog website 

### PR DESCRIPTION
**Pull request by BranBayou**
- First I created 2 pages index.html and about.html.
- I made these two pages responsive by building for the desktop version and mobile version.
- I put links for the home page and about page to make the website responsive when the user clicks the logo or the about nav link.
- I created a mobile hamburger menu when the screen gets small than 768px.]
- I created a dynamically generated section for featured blogs using a javascript array object.
- I checked for linter errors.
- I recorded a loom video to present my capstone project website: https://www.loom.com/share/916b01dc78ad4ece8bb25dd1148ac28d

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -91,6 +91,9 @@ <h5>knowledge-based system</h5>
           <h5>Robotics</h5>
           <p>Robotics is a branch of engineering that involves the conception, design, manufacture and operation of robots.</p>
         </div>
+        <div class="show-all-blogs-btn">
+          <button class="show-all-blogs-bttton mt-5" type="button">Join b personal blog</button>
+        </div>
```
Comment (Ibizugbe) [inline index.html:96]: Nice job on your landing page👍 But let's ensure to stick to the design by taking away the top dark bar on the mobile screen😉 the top dark bar should only be present on the desktop screen as displayed in the design😉
also, kindly take away the white space and ensure the background image is uniform just as presented in the design😉
NO TOP BAR

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @BranBayou 👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Linters are passing👍
- [x] Github flow practice has been used👍
- [x] An array of object containing the information of the various speakers has been created 👍.
- [x] The speakers section is generated dynamically using JavaScript👍.
- [x] Video presentation looks great👍


## Required Changes ♻️

- [ ] Your about page looks great on desktop👍 but let's stick to the design even on mobile😉 your about page looks very different from the design😉 Kindly ensure to use the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) as a guideline and make the page responsive😉 don't forget we are required to stick to the given [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) in this project😎
your about mobile

- [ ] Kindly use the semantic `hx` hierarchy rule when chosing the header to use all through your project😉 Use `h1` for the intro or main header of the page, use `h2` for the main header of each section and use `h3`, `h4` etc as you go down the hierarchy. this helps screen readers to understand the position of a heading. for instance, a `h2` under a `h1` shows that the `h2` is a heading under the `h1` heading. Kindly make the necessary change all through your project😉
semantic respect

- [ ] Just as shown in the image above, kindly ensure all your `alt` attribute contents are filled with a descriptive text that gives information about the image😉 This makes your images meaningful to screen readers and also give your users an idea of what the image is about when the network is slow or the link is broken😉


_Check the comments under the review._

## Optional suggestions
-  _no optional suggestion_
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -62,35 +62,38 @@ <h4 class="w-50">Debretabor Ehiopia</h4>
   </section>
   <section class="main-blog h-100">
     <div class="container h-100 d-flex flex-column justify-content-center align-items-center">
-      <h2 class="pro-intro mt-5">Recent Blogs</h2>
+      <h1 class="pro-intro mt-5">Recent Blogs</h1>
```
Comment (huseyinbiyik) [inline index.html]: As the previous reviewer requested, please set your hx element hierarchically. Kindly use the h1 element on the page once. h1 refers to the most important element on the page. You can check this link to learn more: https://techstacker.com/seo-only-use-h1-elements-once-per-page/

hx elements used for SEO and accessibility of your page. It is not about the size of the text element. So the most important element is h1 and used it once, and it goes down h2,h3...h6. You can use h2 for **each** section title: 
-index.html
Recent Blogs
Featured Blogs
Partners
-about.html
Creative Commons b personal blog 2023 Logo
See tha past bloges by b personal blog

You can also check this [project](https://github.com/huseyinbiyik/world-tomato-forum) to understand more.

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @BranBayou 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
tenor

 ## Highlights
- Linter set up well and checks pass 🟢
- .gitignore used to ignore requested files 📁
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝


 ## Required Changes ♻️
- As the previous reviewer requested, please make your web page responsive and follow the design guideline. Below, you can find the issues.
- - The about page still has a horizontal overflow.
Screen Recording 2023-03-22 at 9 19 31 PM

- - Please add space around the images like on the design guidelines.
Screenshot 2023-03-22 at 9 24 57 PM
Screenshot 2023-03-22 at 9 20 20 PM

- - Kindly fix the section title and make sure it matches the design guideline.
Screenshot 2023-03-22 at 9 27 03 PM
Screenshot 2023-03-22 at 9 25 09 PM

- - Please fix the overflowed logo. Some of it are out of the screen like below.
Screenshot 2023-03-22 at 9 27 23 PM


_Check the comments under the review._

## Optional suggestions
- Kindly make sure you can navigate from about page to home page on mobile version/mobile navigation. In this way, you will enhance your UX.
- Please stick to the design and align your top black nav vertically aligned. There is more space under texts/icons 👍 
Screenshot 2023-03-22 at 9 28 13 PM
- Please use descriptive branch names instead of "feature" (e.g.: refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. [Check](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) more to learn more about this.


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @BranBayou  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [ ] Your page styling looks good as per the previous request :+1:, but it's important to note that there is still a horizontal scroll on the about page when it's viewed on the mobile version. This can impact the user experience and may result in content being cut off or inaccessible. Therefore, it's important to fix this issue to ensure proper functionality and usability on all devices :+1:.


https://user-images.githubusercontent.com/105117832/227188790-cdd3b371-c6fb-4b24-b3fb-d8b4c3c00c16.mp4

- [ ] Your page layout looks good, but as previously mentioned, it's important to match it as closely as possible with the template. To achieve this, it would be better to add some space to each side of the images on the about page in the mobile version. This will help to align the images with the template and enhance the overall visual appeal of the page.

**Your design**

Screenshot from 2023-03-23 06-25-50

**Template**

Screenshot from 2023-03-23 06-25-17

- [ ] As requested by the previous reviewer kindly fix the section title on the about page on the mobile version and make sure it matches the design guideline because now everything is overlapping.

**Your design**

Screenshot from 2023-03-23 06-29-41

**Template**

Screenshot from 2023-03-23 06-33-37

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @BranBayou, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**